File: rmt/src/test/scala/de/flower/rmt/ui/player/NavigationPanelTest.scala
package de.flower.rmt.ui.player

import de.flower.rmt.test.AbstractWicketIntegrationTests
import org.testng.annotations.Test
import de.flower.rmt.ui.common.page.INavigationPanelAware

/**
 * 
 * @author flowerrrr
 */

class NavigationPanelTest extends AbstractWicketIntegrationTests {

    @Test
    def testRender() {
        wicketTester.startComponentInPage(new NavigationPanel(new INavigationPanelAware {
            def getActiveTopBarItem = "home"
        }))
        wicketTester.dumpPage()
    }


}

File: rmt/src/test/java/de/flower/rmt/ui/player/NavigationPanelTest.java
package de.flower.rmt.ui.player;

import de.flower.rmt.test.AbstractRMTWicketMockitoTests;
import de.flower.rmt.ui.common.page.INavigationPanelAware;
import org.testng.annotations.Test;



/**
 * @author flowerrrr
 */

public class NavigationPanelTest extends AbstractRMTWicketMockitoTests {

    @Test
    public void testRender() {
        wicketTester.startComponentInPage(new NavigationPanel(new INavigationPanelAware() {
            @Override
            public String getActiveTopBarItem() {
                return "home";
            }
        }));
        wicketTester.dumpPage();
    }
}